frontend/src/components/footer.css

```diff
@@ -0,0 +1,115 @@
+* {
```

Comment:
This was moved to `components/common/footer/index.css`; please move any relevant CSS there and delete this file.

Also:
* kebab-case throughout
* Remove styling from any native HTML elements